runtime/tr.source/trj9/build/toolcfg/host/ztpf.mk

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
start date?

Reply:
There should only be one copyright date.